src/ast/rewriter/expr_safe_replace.cpp

```diff
@@ -71,6 +55,11 @@ void expr_safe_replace::operator()(expr* e, expr_ref& res) {
         res = e;
         return;
     }
+
+    for (auto &[src,dst] : m_subst) {
```

Comment:
Are you also going to update our build chain from C++11 now that I updated the Centos build?

Reply:
nothing would make me happier than moving Z3 to C++20! 😁
Ok, fine, C++17 for now.

Reply:
I've removed the non-C++11 stuff from this commit + a few little tweaks for.. a nice 50% speedup 🚀